scripts/build-no-chunks.js

```diff
@@ -0,0 +1,15 @@
+// https://github.com/facebook/create-react-app/issues/5306#issuecomment-433425838
+// I am including this so that when we build, we get a single, "main.xxxx.js" file
+// in our build, which we can include in website.
+
+const rewire = require("rewire");
+const defaults = rewire("react-scripts/scripts/build.js");
+let config = defaults.__get__("config");
+
+config.optimization.splitChunks = {
+  cacheGroups: {
+    default: false
+  }
+};
+
+config.optimization.runtimeChunk = false;
```

Comment:
This is pretty neat - just curious, how did we figure this out? I don't recall if my CRA onboarding proj had this issue/needed this rewire

Reply:
It's not usually an issue, because if you're just running create react app, or running the build per usually, it is all wired up for code splitting.

The problem however, is that the way we include this into `website` is copy / paste a single built js and css file: https://github.com/swiftype/website/pull/8003/files. So both the script that copies the files over to `website`, AND website itself is not set up to use chunks or code splitting. So when I attempt to copy the files after upgrading `react-scripts`, there were some errors.

Does that make sense?

Reply:
Ahhh got it! That makes total sense. Thanks for the education! 👩‍🎓